packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_Element.hpp

```diff
@@ -90,7 +90,13 @@ void ReferenceQuad4(scalar_2d_array_t & elementMatrix) {
   }
 }
 
-
+// RHS vector for the reference quad.
+// This can be thought of as a unit source being equally distributed to the
+// 4 nodes of the quad.
+void ReferenceQuad4RHS(Teuchos::Array<Scalar>& rhs) {
```

Comment:
OK for now, but think about replacing all Teuchos::Array with std::vector or Kokkos::View.